### DISCUSSION TITLE
How to disable UTF-8 warning in Powershell 7.2.0

### DISCUSSION THREAD

**Discussion by afgriffiths** [fastlane/fastlane#19639] [Q&A]
I am on Windows 11 and I am using PowerShell 7.2.0. 

Each time I run Fastlane I get the following warning: `WARNING: fastlane requires your locale to be set to UTF-8. To learn more go to https://docs.fastlane.tools/getting-started/ios/setup/#set-up-environment-variables`

Following the link only provides help for Linux and MacOS systems. 

I have set Windows to use UTF-8 as per this stack overflow answer: https://stackoverflow.com/a/57134096/4346886.

The message still appears, how can I fix it?

**Comment by dholeman1** [comment]
This worked for me, ymmv.

Windows users set their environment variables through a modal dialogue. 

For Windows 10 (hopefully similar for 11): On the Windows taskbar, right-click the Windows icon and select  System | About | Advanced Settings | Environment Variables. 

Create two new System Environment Variables:

Variable name: LC_ALL, Variable value: en_US.UTF-8
Variable name: LANG, Variable name: en_US.UTF-8

Then Apply, then OK, then reboot. 

>> Following the link only provides help for Linux and MacOS systems.
